helm/prometheus/locals.tf

```diff
@@ -8,12 +8,29 @@ locals {
   storage_class = merge(local.storage_class_defaults, var.storage_class)
 
   prometheus_helm_defaults = {
-    "server.persistentVolume.storageClass"       = lookup(local.storage_class, var.cloud_provider, "gp2")
-    "server.persistentVolume.size"               = "20Gi"
-    "server.retention"                           = "7d"
-    "alertmanager.persistentVolume.storageClass" = lookup(local.storage_class, var.cloud_provider, "gp2")
```

Comment:
making sure i understand - are the alertmanager.* keys removed bc alertmanager is turned off?

Reply:
yes, I don't think alert manager is useful in our existing setups nor the influxdb setup.  I can't see a way for us to pass configuration in, and an alertmanager that alerts to no where is kind useless.